form-design-intro.rst

```diff
@@ -0,0 +1,49 @@
+Intro to Forms in ODK
+========================
+
+Forms used in the ODK ecosystem are XML documents following the `ODK XForms specification <https://opendatakit.github.io/xforms-spec/>`_, a subset of the `W3C XForms specification <https://www.w3.org/TR/xforms/>`_. Most ODK tools use the `ODK Javarosa <https://github.com/opendatakit/javarosa>`_ library to render forms. Form transfer between ODK components is governed by the :doc:`openrosa` API.
```

Comment:
Maybe we should link some [example forms](https://github.com/opendatakit/sample-forms) here to give an initial insight to a new user.
> To get an initial insight about forms in ODK ecosystem, you may see these examples.

Reply:
That's an interesting idea, but I wonder if it would make things even more confusing...
I'll think about how that might be done without overwhelming someone.

Reply:
We can add a note further which will be linked to a detailed form design section which can be added later.
> These examples are just to provide a basic idea. Details are discussed here.